src/ep.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "cortexApi": {
-    "path": "http://localhost:8080/cortex",
```

Comment:
Just want to be sure if this change was intentional? We're pointing to these values OOTB now?

Reply:
I think we should point to '/cortex' OOTB. There is no reason to keep localhost when webpack devserver can proxy the calls.